src/components/Post.js

```diff
@@ -0,0 +1,91 @@
+import { useState } from "react";
+import axiosInstance from "../api/axios";
+import img1 from "../images/img1.jpg";
+import { useNavigate } from "react-router-dom";
+
+const Post = () => {
+  const [blog, setBlog] = useState({ title: "", content: "", author: "" });
+  const [postErr, setPostErr] = useState("")
+
+
+  const handleBlogDetails = (e) => {
+    e.preventDefault();
+    setBlog((current) => ({ ...current, [e.target.name]: e.target.value }));
+  };
+
+  let navigate = useNavigate();
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const result = await axiosInstance.get("/", blog, {
+
+      });
+      console.log(result)
+      setBlog();
```

Comment:
Why?

Reply:
still working on this, I havent gotten the desired result because I want pass in the bear token from authenticated users to be able to post Blog.